app/templates/partials/questions/repeatinganswer.html

```diff
@@ -1,9 +1,36 @@
 <div class="question question--repeatinganswer" id="{{question.id}}">
 
-  <h2 class="question__title neptune">{{question_title}}</h2>
+  <h2 class="question__title neptune">
+    {%- if question_number -%}
+      <span>{{question_number}}. </span>
+    {% endif %}
+    {{question_title}}
+  </h2>
 
   {{question_description|safe}}
 
+  {%- if question.schema_item.guidance -%}
```

Comment:
I don't think this is necessary...there's updates to display guidance in my summary branch.